index.bs

```diff
@@ -317,6 +340,7 @@ events are also not excluding inputs.
     readonly attribute long value;
     readonly attribute boolean hadRecentInput;
     readonly attribute DOMHighResTimeStamp lastInputTime;
+    readonly attribute FrozenArray&lt;LayoutShiftAttribution&gt; sources;
```

Comment:
Hmm I don't think we want a FrozenArray because there are Nodes inside the object.